addons/cert.py

```diff
@@ -120,6 +120,8 @@ def exp05(data):
             function = token.astOperand1.function
             arguments = cppcheckdata.getArguments(token.previous)
             for argnr, argvar in function.argument.items():
+                if not arguments:
```

Comment:
Just for my understanding: I do not see where `arguments` is changed inside the loop. Why is it not checked outside the loop?

Reply:
Oh, sure, thanks